packages/react-dev-utils/failFast.js

```diff
@@ -254,6 +254,25 @@ function renderAdditional() {
   additionalReference.appendChild(span)
 }
 
+function removeNextBr(parent, component) {
+  while (component != null && component.tagName.toLowerCase() !== 'br') {
+    component = component.nextSibling
+  }
+  if (component != null) {
+    parent.removeChild(component)
+  }
+}
+
+function absolutifyCode(component) {
+  for (let c of component.childNodes) {
```

Comment:
Forgive my ignorance, but is there an issue with `position: absolute` or `childNodes` in IE9?
I'm not sure if you're stating something or pointing out an issue.